rebasehelper/tests/test_output_tool.py

```diff
@@ -68,39 +68,51 @@ def results_store(self, workdir):
 
         return rs
 
-    def get_expected_text_output(self):
+    def get_expected_text_output(self, workdir):
         expected_output = """\
 Success
+Rebase helper results are located in {}
 
-Summary information about patches:
-Patch mytest2.patch [deleted]
+Downstream Patches
+==================
+Rebased patches are located in rebase-helper-results/rebased-sources
+Legend:
+[-] = already applied, patch removed
+[*] = merged, patch modified
+[!] = conflicting or inapplicable, patch skipped
+[ ] = patch untouched
+* mytest2.patch                            [-]
 
-Old (S)RPM packages:
----------------------
-SRPM package(s): are in directory . :
+RPMS
+====
+
+Old packages
+------------
+
+Source packages and logs are in directory rebase-helper-results/old-build/SRPM:
 - test-1.2.0-1.src.rpm
-RPM package(s): are in directory . :
+- logfile1.log
+- logfile2.log
+
+Binary packages and logs are in directory rebase-helper-results/old-build/RPM:
 - test-1.2.0-1.x86_64.rpm
 - test-devel-1.2.0-1.x86_64.rpm
-Available Old logs:
 - logfile1.log
 - logfile2.log
 
-New (S)RPM packages:
----------------------
-SRPM package(s): are in directory . :
+New packages
+------------
+
+Source packages and logs are in directory rebase-helper-results/new-build/SRPM:
 - test-1.2.2-1.src.rpm
-RPM package(s): are in directory . :
+- logfile3.log
+- logfile4.log
+
+Binary packages and logs are in directory rebase-helper-results/new-build/RPM:
 - test-1.2.2-1.x86_64.rpm
 - test-devel-1.2.2-1.x86_64.rpm
-Available New logs:
 - logfile3.log
-- logfile4.log
-=== Checker Results from checker(s) results ===
-Following files were moved
-/usr/sbin/test
-/usr/sbin/test2
-See for more details pkgdiff"""
+- logfile4.log""".format(workdir)
```

Comment:
Since the string is really long, I would prefer to use keyword argument(s) in `format()`, so `.format(workdir=workdir)` and `Rebase helper results are located in {workdir}`.